plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -309,7 +309,7 @@ function hasValidSqlComment(query: string): boolean {
 }
 
 export function addSqlCommenterComment(span: Span, query: string): string {
-  if (query.length === 0) {
+  if (query.length === 0 || typeof query !== 'string') {
```

Comment:
```suggestion
  if (typeof query !== 'string' || query.length === 0) {
```
in case `query` is undefined you would not reach the typeof check